code/modules/hydroponics/seeds.dm

```diff
@@ -1555,7 +1570,7 @@
 	yield = 3
 	potency = 0
 
-/datum/seed/apple/wood
+/datum/seed/woodapple
```

Comment:
@Skullyton Ah, fuck. I unthinkingly removed the inheritance here. See, this is why you should point out missing stuff on the latest code.